src/components/common/Pagination.tsx

```diff
@@ -1,7 +1,9 @@
 'use client';
 
 import PaginationButton from '@/components/page/wikilist/PaginationButton';
-import { useState, useEffect } from 'react';
+import { FaAngleLeft } from 'react-icons/fa6';
+import { FaAngleRight } from 'react-icons/fa6';
```

Comment:
아이콘이 두개라서 바로 이해할 수 있긴 한데 이렇게 하면 더 눈에 잘 들어올 것 같습니다 :)
```suggestion
import { FaAngleLeft as PaginationPrev } from 'react-icons/fa6';
import { FaAngleRight as PaginationNext } from 'react-icons/fa6';
```

```tsx
<PaginationPrev />
<PaginationNext />
```

Reply:
좋은 방법 같습니다! 바로 반영해야겠네요